test/test_artifacts/autogluon.test.Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG COSMOS_IMAGE
+FROM $COSMOS_IMAGE as base
+
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+ARG AUTOGLUON_VERSION="0.8.2"
```

Comment:
Why are we explicitly tagging the version of auto gluon

Reply:
So When SageMaker Distribution launches a new image, do you expect that to have AutoGluon 0.8.2 in it?

Reply:
If we don't tag it and you want to perform a patch of an existing container while new version of AG is out, that would cause AG version being updated in the container too causing unexpected behavior. I think it would be better if we tag it and manually update it whenever AG does a new release

Reply:
So the problem is that , if we don't update AG then this will likely to happen
* AG might have some dependencies which are outdated.
* These outdated dependencies might prevent us from updating marquee packages 

We should come up with a long term solution for this